scripts/home.js

```diff
@@ -12,10 +8,8 @@ const config = require('../lib/config')
  */
 function index(_req, res) {
   const authUrl = {}
-  for (const [env, baseurl] of Object.entries(config.baseUrls)) {
-    authUrl[env] = baseurl
-      ? `${baseurl}/v1/oauth/authorize?response_type=code&client_id=${clientId}&scope=${scopes}&redirect_uri=${hostname}/callback&nonce=randomnonce&state=${env}`
-      : undefined
+  for (const [env, ] of Object.entries(config.baseUrls)) {
```

Comment:
nit: we can use `Object.keys(config.baseUrls)` if we just want the keys, and the `.forEach` array method for readability